packages/python/plotly/plotly/tests/test_core/test_graph_objs/test_performance.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import time
+from unittest import TestCase
+import pytest
+import numpy as np
+import plotly.graph_objs as go
+import plotly.io as pio
+from plotly.tests.b64 import b64, _b64
+
+
+def test_performance_b64():
+    rand_arr_1 = np.random.random(100000)
```

Comment:
These are `float64` numbers by current numpy default.
How about clarifying the test name to be `test_performance_b64_float64` similar to `unit8` one.